psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -2015,7 +2016,13 @@ void DFHelper::transform() {
 
     // transformations complete, time for metric contractions
```

Comment:
What is the transform function trying to accomplish? The function isn't properly documented, and I can't review the code without that.

Reply:
This transform function converts 3-index integrals with AO functions (pq|Q) to those with MO functions. The details of how they should be transformed are configured via other functions like `add_space()` and `add_transformation()`, and `transform()` performs the actual transformation with the configurations defined above.